crates/forge/tests/e2e/common/runner.rs

```diff
@@ -12,13 +12,15 @@ pub(crate) fn runner() -> SnapboxCommand {
     snapbox
 }
 
-pub(crate) fn setup_package(package_name: &str) -> TempDir {
+pub(crate) static BASE_FILE_PATTERNS: &[&str] = &["**/*.cairo", "**/*.toml", "**/*.txt"];
+
+pub(crate) fn setup_package_with_file_patterns(
+    package_name: &str,
+    file_patterns: &[&str],
+) -> TempDir {
```

Comment:
Can't we just append `"**/*.json"` to patterns in `fn setup_package`? No need to complicate the logic imo

Reply:
I guess it's okay then